kolibri/core/bookmarks/models.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import models
+from morango.models import UUIDField
+
+from kolibri.core.auth.models import FacilityUser
+
+
+class Bookmark(models.Model):
+
+    id = UUIDField(primary_key=True, default=uuid.uuid4)
```

Comment:
Can skip the id field, as this is the default if you have inherited from `AbstractFacilityDataModel` above.